src/test/scala/io/iohk/ethereum/ledger/LedgerTestSetup.scala

```diff
@@ -479,7 +478,7 @@ trait EphemBlockchain extends TestSetupWithVmAndValidators with MockFactory {
 trait CheckpointHelpers {
   private val sampleCheckpoint = ObjectGenerators.fakeCheckpointGen(3, 3).sample.get
 
-  def getCheckpointBlock(parent: Block, difficulty: BigInt, checkpoint: Checkpoint = sampleCheckpoint): Block =
```

Comment:
@LeonorLunatech what is the reason for this change?

Reply:
The parameter `difficulty` was not being used

Reply:
We have several warnings being output when the code compile, and we are trying to take care of them. This was one of them